.eslintrc

```diff
@@ -0,0 +1,77 @@
+{
+  "root": true,
+  "env": {
+    "amd": true,
+    "browser": true,
+    "es6": true,
+    "jest": true,
+    "node": true
+  },
+  "globals": {
+    "globalThis": false
+  },
+  "parserOptions": {
+     "project": "./tsconfig.json"
+  },
+  "extends": [
+    "airbnb-typescript/base",
+    "prettier/@typescript-eslint"
+  ],
+  "plugins": [
+    "@typescript-eslint",
+    "prettier"
+  ],
+  "rules": {
```

Comment:
This whole section is a merging of `locker/.eslintrc.js` and `locker/packages/@locker/.eslintrc.js`, deferring to the latter.

Reply:
This is no longer true.